Makefile

```diff
@@ -24,13 +23,25 @@ CONTAINER_ENGINE?=docker
 default: all
 all: build-base build-sss
```

Comment:
Is it worth looping in the `test` target here?

Reply:
It's pulled in as a dependency by build-base.